doc/sphinxext/update_credit_rst.py

```diff
@@ -3,6 +3,9 @@
 Run ./tools/dev/update_credit_json.py first to get the latest PR JSON files.
 """
 
+# Authors: The MNE-Python contributors.
+# License: BSD-3-Clause
+# Copyright the MNE-Python contributors.
 import glob
```

Comment:
FYI @drammock it would be easy enough now I think to make sure there is a newline here before code if we want it, too.